src/Routers/UsersRouter.js

```diff
@@ -57,6 +57,17 @@ export class UsersRouter extends ClassesRouter {
           const user = response.results[0].user;
           // Send token back on the login, because SDKs expect that.
           user.sessionToken = sessionToken;
+
+          // Remove hidden properties.
+          for (var key in user) {
```

Comment:
just a quick question,
Those fields get stripped only when calling /me, but not when let's say we make an unauthenticated query?

Could we add a test for that?